read.go

```diff
@@ -46,9 +46,9 @@ func (i *UI) read(opts *readOptions) (string, error) {
 				resultErr = fmt.Errorf("failed to read the input: %s", err)
 			}
 
-			resultStr = strings.TrimSuffix(line, LineSep)
+			resultStr = strings.Trim(line, LineSep)
 			// brute force for the moment
-			resultStr = strings.TrimSuffix(line, "\n")
+			resultStr = strings.TrimSuffix(resultStr, "\n")
```

Comment:
Note: I'm not sure of the reasoning for this line so I left it in, but I'm guessing you needed it to get rid of new lines no matter what since they probably weren't working for windows.  Using Trim instead of TrimSuffix should solve this problem, so you may be able to remove this line